node/api/logs.go

```diff
@@ -37,6 +37,7 @@ type ContainerLogOpts struct {
 	Since      time.Duration
 	LimitBytes int
 	Timestamps bool
+	Follow     bool
```

Comment:
we  can not just add Follow option, since the current implementation actually do not support it

Reply:
@cwdsuzhou you mean providers don't support it? Ours definitely can once this field is in the struct.

Reply:
Maybe something wrong with mysql implementation. But when I change some logic of `vk`, it works.